src/main/java/com/logentries/net/LogentriesClient.java

```diff
@@ -227,23 +227,8 @@ private SSLContext CreateAndInitSSLContext(String algorithm) throws NoSuchAlgori
 			KeyManagementException {
 
 		SSLContext sslContext = SSLContext.getInstance(algorithm);

```

Comment:
hhhmmmm...yes, I see what you mean - and I see a couple of other issue - oh well - back to the drawing board.